docs/www/buildspec.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 NumberFour AG.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   NumberFour AG - Initial API and implementation
+#
+set -e 
+echo "Terminate running but possibly outdated asciispec server..."
+asciispec term
```

Comment:
I have a request: I introduced these two lines above into all build.sh scripts, I remember, but since the asciispec-updater script is doing the termination now, I think it's better to remove these two lines from all build.sh scripts (five in N4JS repo, but maybe more)  again, because they just increase the execution times. -- Could you do that?